libr/include/r_cons.h

```diff
@@ -415,7 +415,7 @@ typedef struct r_cons_t {
 #define Color_BGCYAN     "\x1b[46m"
 #define Color_BLUE       "\x1b[34m"
 #define Color_BGBLUE     "\x1b[44m"
-#define Color_GRAY       "\x1b[38m"
+#define Color_GRAY       "\x1b[38;5;8m"
```

Comment:
noes. gray != bgray

Reply:
<img width="308" alt="screen shot 2018-02-18 at 04 05 02" src="https://user-images.githubusercontent.com/917142/36347800-eb9a5368-1460-11e8-82df-66e1e730a549.png">

Reply:
also you may use \033[30m to make this thing work as expected.. i would go to support "simple" ansi codes

Reply:
black? Like this 
![screenshot 0bb582c80646cd5bcc668434f96b6719](https://user-images.githubusercontent.com/4048120/36435744-c9420a5a-1662-11e8-9165-9c006d6df8f0.png)